<!-- move base -->
<launch>
  <master auto="start"/>
    <arg name="visualize" default="true"/>

<!-- <param name="use_sim_time" value="true" /> -->

  <node pkg="move_base" type="global_planner_node" respawn="false" name="global_planner_node" output="screen">

  		<remap from="/cmd_vel" to="/mobile_base/commands/velocity"/>

    <rosparam file="$(find mpc_local_planner_examples)/cfg/diff_drive/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find mpc_local_planner_examples)/cfg/diff_drive/global_costmap_params.yaml" command="load" />


		<param name="base_global_planner" value="global_planner/GlobalPlanner" />
		<param name="planner_frequency" value="1.0" />
		<param name="planner_patience" value="5.0" />

    <!-- <param name="global_costmap/global_frame" value="map"/>
    <param name="global_costmap/robot_base_frame" value="base_footprint"/>
    <param name="global_costmap/laser_scan_sensor/sensor_frame" value="base_laser_link"/>
    <param name="global_costmap/laser_scan_sensor/topic" value="/scan"/>     -->
  </node>

   <!-- <node if="$(arg visualize)" name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find move_base)/rviz/turtlebot_tracker.rviz" /> -->
</launch>
